modules/openideal_statistics/src/Plugin/Block/Charts.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Drupal\openideal_statistics\Plugin\Block;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\Core\Entity\EntityTypeManager;
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a 'Charts' block.
+ *
+ * @Block(
+ *  id = "openideal_statistics_charts_block",
+ *  admin_label = @Translation("Charts block"),
+ * )
+ */
+class Charts extends BlockBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * Entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManager
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Json serialization service.
+   *
+   * @var \Drupal\Component\Serialization\Json
+   */
+  protected $serializer;
+
+  /**
+   * {@inheritDoc}
+   */
+  public function __construct(
+    array $configuration,
+    $plugin_id,
+    $plugin_definition,
+    EntityTypeManager $entityTypeManager,
+    Json $json
+  ) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->entityTypeManager = $entityTypeManager;
+    $this->serializer = $json;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('entity_type.manager'),
+      $container->get('serialization.json')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $users = $this->entityTypeManager->getStorage('user')->loadMultiple();
```

Comment:
1) Should we filter blocked users?
2) Do we need to include all roles including admins and managers?